### PR TITLE
Converted all network thread logs to ocp diag format

### PR DESCRIPTION
**Pull request by dhawkes**
Internal reference: 274516894, 274517465

### CODE REVIEW

**Comment by dhawkes** [comment]
Current dependencies on/for this PR:
* main
  * **PR #10** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/10" target="_blank">Graphite</a> 
    * **PR #12** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/12" target="_blank">Graphite</a> 
      * **PR #11** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/11" target="_blank">Graphite</a> 
        * **PR #13** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/13" target="_blank">Graphite</a> 
          * **PR #14** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/14" target="_blank">Graphite</a> 
            * **PR #15** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/15" target="_blank">Graphite</a> 
              * **PR #16** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/16" target="_blank">Graphite</a> 
                * **PR #17** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/17" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/17?utm_source=stack-comment).

**Comment by dhawkes** [comment]
Current dependencies on/for this PR:
* main
  * **PR #10** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/10" target="_blank">Graphite</a> 
    * **PR #12** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/12" target="_blank">Graphite</a> 
      * **PR #11** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/11" target="_blank">Graphite</a> 
        * **PR #13** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/13" target="_blank">Graphite</a> 
          * **PR #14** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/14" target="_blank">Graphite</a> 
            * **PR #15** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/15" target="_blank">Graphite</a> 
              * **PR #16** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/16" target="_blank">Graphite</a> 
                * **PR #17** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/17" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/17?utm_source=stack-comment).

File: src/worker.h [hunk 1]
```diff
@@ -476,6 +476,8 @@ class NetworkThread : public WorkerThread {
   // Calculate worker thread specific bandwidth.
   virtual float GetDeviceCopiedData() { return GetCopiedData() * 2; }
 
+  string GetThreadTypeName() { return "Network IO Thread"; }
```
Comment (jcgao1) [inline src/worker.h:479]: Are these Getters used anywhere?

Comment (dhawkes) [reply src/worker.h:479]: Yea these are used in the AddLog, AddProcessError, and AddDiagnosis functions in the parent class in order to append the thread name and id to the start of each artifact message.

**Review by jcgao1 — APPROVED** [review]

**Review by jcgao1 — APPROVED** [review]

**Comment by dhawkes** [comment]
@dhawkes started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/17).

**Comment by dhawkes** [comment]
@dhawkes started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/17).

**Comment by dhawkes** [comment]
@dhawkes merged this pull request with [Graphite](https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/17).

**Pull request merged**